LQResultProducer.py

```diff
@@ -5642,9 +5649,9 @@ def MakeBasicPlot(recovariable,xlabel,presentationbinning,selection,weight,FileD
 
 
 	MCStack=BeautifyStack(MCStack,Label)
-	hs_rec_Signal.Draw("HISTSAME")
-	if 'final' not in tagname:
-		hs_rec_Signal2.Draw("HISTSAME")
+	#hs_rec_Signal.Draw("HISTSAME")
```

Comment:
I don't think you want the signal commented out anymore, do you? We should have signal samples now, can start getting them on the plots.

Reply:
True! Added it in.